tests/contrib/requests/test_requests.py

```diff
@@ -492,3 +492,14 @@ def test_analytics_integration_on_using_pin_default(self):
         self.assertEqual(len(spans), 1)
         s = spans[0]
         self.assertEqual(s.get_metric(ANALYTICS_SAMPLE_RATE_KEY), 1.0)
+
+    @BaseTracerTestCase.run_in_subprocess(env_overrides=dict(DD_SERVICE="mysvc"))
```

Comment:
I think we should have a test where we have both the `DD_SERVICE` and `DD_SERVICE_NAME` provided. Folks could definitely run into that case when they migrate.

I don't think this is relevant for all integrations, just the ones where you have
```
config.get_service(default=get_env(...))
```

Reply:
`config.get_service(...)` deals with also checking `DD_SERVICE_NAME` (and `DATADOG_SERVICE_NAME`). Although I'm not sure if I added a test for `config.get_service()` to ensure this. So I'll make sure to do that.

Do you think that's sufficient or should we still have tests on each integration to ensure the compatibility?

Reply:
Yeah it might be covered by calling `config.get_service()` with one/both of the env vars provided.